addons/mail/models/mail_guest.py

```diff
@@ -52,19 +59,19 @@ def _get_timezone_from_request(self, request):
     def _update_name(self, name):
         self.ensure_one()
         name = name.strip()
-        if name == "":
-            raise UserError(_("Guest's name cannot be empty."))
+        if not self.validate_guest_name(name):
+            raise UserError(_("Invalid guest name."))
```

Comment:
This actually made the message less useful than before.

Reply:
I think I'll remove the validation function for now, and do the checks here instead